tests/test_cli.py

```diff
@@ -249,3 +249,78 @@ def test_folders_default():
         args = pyclean.cli.parse_arguments()
 
     assert not args.folders
+
+
+def test_git_clean_option_available_when_git_installed():
+    """
+    Is `--git-clean` option available when Git is installed?
+    """
+    if not pyclean.cli.is_git_available():
+        pytest.skip('Git not installed')
+
+    with ArgvContext('pyclean', '.', '--git-clean'):
+        args = pyclean.cli.parse_arguments()
+
+    assert args.git_clean
+
+
+@pytest.mark.parametrize('option', ['--git-clean', '-g'])
+def test_git_clean_option_short_and_long(option):
+    """
+    Is `--git-clean` option available in both short and long form?
+    """
+    if not pyclean.cli.is_git_available():
+        pytest.skip('Git not installed')
```

Comment:
Can this (or a functionally equivalent code) be added as test function decorator?